src/FakeItEasy/Configuration/IAfterCallSpecifiedConfiguration.cs

```diff
@@ -3,8 +3,9 @@ namespace FakeItEasy.Configuration
     /// <summary>
     /// Lets you set up expectations and configure repeat for the configured call.
     /// </summary>
-    public interface IAfterCallSpecifiedConfiguration
-        : IRepeatConfiguration

```

Comment:
Currently, it doesn't seem very useful, but I think it makes sense to have it. If we removed it and returned `IRepeatConfiguration` directly, we couldn't add anything else after a call configuration without making a breaking change to the API. Currently, we just need to add another base interface if we want to do that (which, technically, is also a breaking change, but the users aren't supposed to implement that interface anyway)

That being said, I think the name is confusing; it should probably be called `IAfterCallConfiguredConfiguration` (which doesn't sound very good, but is more accurate).